.eslintrc.json

```diff
@@ -0,0 +1,127 @@
+{
+  "env": {
+    "browser": true,
+    "es2020": true,
+    "jest": true
+  },
+  "extends": [
+    "plugin:@typescript-eslint/recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "plugin:import/typescript",
+    "plugin:jsx-a11y/strict",
+    "prettier"
+  ],
+  "settings": {
+    "import/resolver": {
+      "typescript": {}
```

Comment:
Bruke den relativ urlen til `tsconfig.json` ? usikker

Reply:
Tror ikke nødvendig her heller

Reply:
Betyr det at `"typescript": {}` kan fjernes?

Reply:
Jeg trenger den ihvertfall lokalt for at linteren skal funke. Hvis du har forslag til alternative måter å løse det på tar jeg gjerne inn det.